src/test/java/Homework21.java

```diff
@@ -0,0 +1,67 @@
+import org.openqa.selenium.By;
+import org.openqa.selenium.Keys;
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.WebElement;
+import org.openqa.selenium.support.ui.ExpectedConditions;
+import org.openqa.selenium.support.ui.WebDriverWait;
+import org.testng.Assert;
+import org.testng.annotations.Test;
+import java.time.Duration;
+
+
+
+public class Homework21 extends BaseTest {
+    String playlistName = "Deleted Playlist";
+
+
+    @Test
+
+    public void renamePlaylist() throws InterruptedException {
+        String updatedPlaylistMsg = "Updated playlist \"Deleted Playlist.\"";
+
+
+        doubleClickPlaylist();
+        enterNewPlaylistName();
+
+
+        Assert.assertEquals(getRenamePlaylistSuccessMsg(), updatedPlaylistMsg);
+    }
+
+
+    private String getRenamePlaylistSuccessMsg() {
+        WebElement notification = wait.until(ExpectedConditions.visibilityOfElementLocated(By.cssSelector("div.success.show")));
+        return notification.getText();
+    }
+
+    private void enterNewPlaylistName() {
+        WebElement playlistInputField = wait.until(ExpectedConditions.visibilityOfElementLocated(By.cssSelector("[name='name']")));
+        playlistInputField.sendKeys(Keys.chord(Keys.CONTROL, "A"));
+        playlistInputField.sendKeys(Keys.BACK_SPACE);
+        playlistInputField.sendKeys(playlistName);
+        playlistInputField.sendKeys(Keys.ENTER);
+
+    }
+
+    private void doubleClickPlaylist() {
+        WebElement playlistElement = wait.until(ExpectedConditions.visibilityOfElementLocated(By.xpath("//*[@id='playlists']/ul/li[3]")));
+        actions.doubleClick(playlistElement).perform();
+    }
+}
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
```

Comment:
**Please delete the empty line!** An Empty Line is a "Code Smell"
The short answer is, _They’re more an annoyance than a problem.They won’t actually break anything—which is why your code continues to run—but they make it awkward to read and change your code, especially if you’re dealing with multiple line breaks per line of real code. They also add just a tiny bit of size to your files, which is nice to reduce._

Reply:
I don't know how to do it. I lost this test from homewok-22. How to fix all mistakes here in GitHub?

Reply:
I resubmitted it. Check please.